src/models/methods/ripplePathFind.ts

```diff
@@ -0,0 +1,31 @@
+import { Amount, LedgerIndex, Path } from "../common";
+import { BaseRequest, BaseResponse } from "./baseMethod";
+
+interface SourceCurrencyAmount {
+    currency: string
+    issuer?: string
+  }
+
+export interface RipplePathFindRequest extends BaseRequest {
+    command: "ripple_path_find"
+    source_account: string
+    destination_account: string
+    destination_amount: Amount
+    send_max?: Amount
+    source_currencies: SourceCurrencyAmount
```

Comment:
It seems like source_currencies should be optional - https://xrpl.org/ripple_path_find.html#ripple_path_find

Reply:
Good catch, thanks